scripts/patchCJS.ts

```diff
@@ -17,7 +17,7 @@ module.exports.parseVueRequest = parseVueRequest;
 */
 
 import { readFileSync, writeFileSync } from 'fs'
-import { bold, red } from 'picocolors'
+import colors from 'picocolors'
```

Comment:
This doesn't look like related to the PR :thinking:

Reply:
`picocolors` does not have named exports. This usage causes the script to fail.